src/navigation/wallet/screens/send/confirm/PayProConfirm.tsx

```diff
@@ -321,14 +335,27 @@ const PayProConfirm = () => {
               onPress={openKeyWalletSelector}
               hr
             />
+            {invoice ? (
+              <RemainingTime
+                invoiceExpirationTime={invoice.expirationTime}
+                setDisableSwipeSendButton={setDisableSwipeSendButton}
+              />
+            ) : null}
             <Amount description={'SubTotal'} amount={subTotal} />
-            <Amount description={'Total'} amount={total} />
+            <Amount description={'Total'} amount={total} hr={!!txp} />
+            {txp ? (
+              <Memo
+                memo={txp.message}
+                onChange={message => updateTxp({...txp, message})}
```

Comment:
In this case, I don't think so. While we could have a unique memo for each individual output, this particular memo is for describing the transaction as a whole (rather than each individual output)